daemon/cluster/tasks.go

```diff
@@ -47,18 +47,19 @@ func (c *Cluster) GetTasks(options apitypes.TaskListOptions) ([]types.Task, erro
 			return nil
 		}
 
-		filters, err := newListTasksFilters(options.Filters, filterTransform)
+		f, err := newListTasksFilters(options.Filters, filterTransform)
 		if err != nil {
 			return err
 		}
 
 		r, err = state.controlClient.ListTasks(
 			ctx,
-			&swarmapi.ListTasksRequest{Filters: filters},
+			&swarmapi.ListTasksRequest{Filters: f},
 			grpc.MaxCallRecvMsgSize(defaultRecvSizeForListResponse),
 		)
 		return err
-	}); err != nil {
+	})
+	if err != nil {
```

Comment:
Mostly to align with other code in this file, and slightly more readable for these long blocks